src/app/src/components/ModalSections/FileUpload.js

```diff
@@ -22,11 +23,13 @@ import {
 } from '../../utils';
 import { useEndpointToastError, useFilePicker } from '../../hooks';
 import { useStartNewBoundaryMutation } from '../../api/boundaries';
+import { ACCEPT_BOTH } from '../../constants';
 
 export default function FileUpload({ PreviousButton }) {
     const toast = useToast();
     const navigate = useNavigate();
     const utilityId = useSelector(state => state.auth.utility.id);
+    const [isLoading, setIsLoading] = useState(false);
```

Comment:
I recommend using the mutation's `isLoading` value as described in a previous comment.

Reply:
[b2bc044](https://github.com/azavea/iow-boundary-tool/pull/175/commits/b2bc04431e7916c7aec85b852c73150a169fbfe8)